DEVELOPMENT.md

```diff
@@ -2,17 +2,15 @@
 
 The proxy is built on the [Buffalo](https://gobuffalo.io/) framework. We chose
 this framework to make it as straightforward as possible to get your development environment up and running.
+However, **you do not need to install buffalo to run Athens**. 
 
-You'll need Buffalo [v0.12.4](https://github.com/gobuffalo/buffalo/releases/tag/v0.12.4) or later to get started on Athens,
+Buffalo provides nice features like a file watcher for your server, so if you'd like to install Buffalo, download [v0.12.4](https://github.com/gobuffalo/buffalo/releases/tag/v0.12.4) or later to get started on Athens,
 so be sure to download the CLI and put it into your `PATH`.
 
 Athens uses [Go Modules](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) for dependency management. You will need Go [v1.11](https://golang.org/dl) or later to get started on Athens.
 
 See our [Contributing Guide](CONTRIBUTING.md) for tips on how to submit a pull request when you are ready.
 
-# Initial Development Environment Setup
-Athens relies on having a few tools installed locally. Run `make setup-dev-env` to install them.
```

Comment:
maybe we keep `make setup-dev-env`, as it has `go get golang.org/x/lint/golint` (love the linter) but omit the [`get_buffalo.sh` script ](https://github.com/gomods/athens/blob/master/scripts/get_dev_tools.sh#L9)

Reply:
@Layoness, our CI actually uses `make setup-dev-env` to install Buffalo and run an end-to-end test with it (see .travis.yml). So I'm a bit hesitant in changing the CI/CD behavior. 

I can either have a new make command that go gets the linter, or maybe just mention it in the README that we use `golang.org/x/lint/golint` and that users should install/run it before submitting a PR (though that's not necessary)

Reply:
Ah. I hadn't thought of that.  the `go get` buffalo was failing for me locally with an error message identical to https://github.com/gobuffalo/buffalo/issues/1439, until I added a `-u`, so removing it as a instruction in DEVELOPMENT.md and asking users to install themselves is 👍, like you've done with asking users to install docker-compose as a dependency.